Automattic/wp-calypso
# My Home: Update the copy in the site setup checklist

#### Changes proposed in this Pull Request

* Improved the copy for the site setup checklist

![image](https://user-images.githubusercontent.com/13596067/126605310-a80eab24-9a1b-4f9a-b20b-3d7f74cba6c3.png)
![image](https://user-images.githubusercontent.com/13596067/126605499-de7c314b-3620-439b-8966-d993fa22e3d9.png)



#### Testing instructions



* Go to My Home
* Verify the text of title, description and action button of incomplete tasks match spreadsheet linked here: pd2qGl-7s-p2
* Verify the task of “Verify the email address for domain“ doesn't appear when done.
* Go to My Profile > Account Settings > Interface language and change your language to non-English. Re-do steps 2-4 to check English copy isn't shown



Close https://github.com/Automattic/wp-calypso/issues/54708
